sdks/python/apache_beam/runners/portability/job_server_test.py

```diff
@@ -25,7 +25,8 @@
 
 class JavaJarJobServerStub(JavaJarJobServer):
   def java_arguments(
-      self, job_port, artifact_port, expansion_port, artifacts_dir):
+      self,
+          job_port, artifact_port, expansion_port, artifacts_dir, jar_cache_dir):
```

Comment:
I think this is causing `test_subprocess_cmd_and_endpoint` to fail since paths don't pass in the cache dir. See for example https://github.com/s21lee/beam/blob/e8322a7968845ba6c08c2b5fd91b676c99309dfa/sdks/python/apache_beam/runners/portability/job_server.py#L158 - I think we shouldn't add java_cache_dir as an arg here

Reply:
Thank you for comments. I will the parameter and the code.